database/src/main/resources/db/migration/V1_129__associated_service_operators_change.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "associated-service-operators"
+  ADD COLUMN timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  ADD COLUMN "user-id" TEXT REFERENCES public.user (id);
```

Comment:
public isn't needed. Would be better to leave it out so the style remains the same